eclipse-collections/src/main/java/org/eclipse/collections/impl/set/sorted/mutable/TreeSortedSet.java

```diff
@@ -679,6 +679,25 @@ public T getLast()
     }
 
     @Override
+    public T getOnly()
+    {
+        Iterator<T> iterator = this.iterator();
+
+        if (!iterator.hasNext())
+        {
+            throw new IllegalStateException("Size must be 1 but was 0");
+        }
+
+        T result = iterator.next();
+        if (iterator.hasNext())
+        {
+            throw new IllegalStateException("Size must be 1 but was greater than 1");
+        }
+
+        return result;
+    }
+
```

Comment:
I think it's ok to call size() method in `TreeSortedSet` and this class wouldn't need to override the method.